### ISSUE TITLE
[ANDROID] NPE Accessing CamperaPosition.zoom when reporting telemetry event

### ISSUE THREAD

**Issue by sddamico** [mapbox/mapbox-gl-native#12474]
Saw this come through in our crashlytics, I do not have reproduction steps unfortunately.

http://crashes.to/s/28d1be17595

```
Fatal Exception: java.lang.NullPointerException: Attempt to read from field 'double com.mapbox.mapboxsdk.camera.CameraPosition.zoom' on a null object reference
       at com.mapbox.mapboxsdk.maps.Transform.getZoom(Transform.java:222)
       at com.mapbox.mapboxsdk.maps.MapGestureDetector.isZoomValid(MapGestureDetector.java:901)
       at com.mapbox.mapboxsdk.maps.MapGestureDetector.sendTelemetryEvent(MapGestureDetector.java:888)
       at com.mapbox.mapboxsdk.maps.MapGestureDetector.access$700(MapGestureDetector.java:42)
       at com.mapbox.mapboxsdk.maps.MapGestureDetector$MoveGestureListener.onMoveBegin(MapGestureDetector.java:442)
       at com.mapbox.android.gestures.MoveGestureDetector.analyzeMovement(MoveGestureDetector.java:154)
       at com.mapbox.android.gestures.MultiFingerGesture.analyzeEvent(MultiFingerGesture.java:98)
       at com.mapbox.android.gestures.ProgressiveGesture.analyzeEvent(ProgressiveGesture.java:46)
       at com.mapbox.android.gestures.MoveGestureDetector.analyzeEvent(MoveGestureDetector.java:135)
       at com.mapbox.android.gestures.BaseGesture.analyze(BaseGesture.java:56)
       at com.mapbox.mapboxsdk.maps.MapGestureDetector.onTouchEvent(MapGestureDetector.java:3034)
       at com.mapbox.mapboxsdk.maps.MapView.onTouchEvent(MapView.java:438)
       at android.view.View.dispatchTouchEvent(View.java:9959)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2705)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2386)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at com.seatgeek.maps.mapbox.SeatGeekMapView.dispatchTouchEvent(SeatGeekMapView.java:804)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2711)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2400)
       at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:416)
       at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1834)
       at android.app.Activity.dispatchTouchEvent(Activity.java:3154)
       at android.support.v7.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:68)
       at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:378)
       at android.view.View.dispatchPointerEvent(View.java:10179)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4650)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4518)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3969)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4022)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3988)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3996)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3969)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4022)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3988)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4117)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3996)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4174)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3969)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4022)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:3988)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:3996)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:3969)
       at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:6459)
       at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:6433)
       at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:6394)
       at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:6593)
       at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:185)
       at android.view.InputEventReceiver.nativeConsumeBatchedInputEvents(InputEventReceiver.java)
       at android.view.InputEventReceiver.consumeBatchedInputEvents(InputEventReceiver.java:176)
       at android.view.ViewRootImpl.doConsumeBatchedInput(ViewRootImpl.java:6541)
       at android.view.ViewRootImpl$ConsumeBatchedInputRunnable.run(ViewRootImpl.java:6616)
       at android.view.Choreographer$CallbackRecord.run(Choreographer.java:871)
       at android.view.Choreographer.doCallbacks(Choreographer.java:683)
       at android.view.Choreographer.doFrame(Choreographer.java:613)
       at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:857)
       at android.os.Handler.handleCallback(Handler.java:751)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6316)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:872)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:762)
```

**Comment by tobrun** [comment]
thanks for the report, this was fixed with https://github.com/mapbox/mapbox-gl-native/pull/12405 and available in v6.3.0

**Comment by LukasPaczos** [comment]
This actually should've been fixed in https://github.com/mapbox/mapbox-gl-native/pull/12042, available in `v6.2.0`. Please let us know if the newest version of the SDK is not working for you @sddamico.
